src/client/pages/Register.jsx

```diff
@@ -1,14 +1,15 @@
 import React, { useContext, useState } from 'react';
 import { Redirect, Link } from 'react-router-dom';
 // eslint-disable-next-line import/no-unresolved
-import { FormularioInput } from '../components/FormularioInput';
-import usuarioContexto from '../state/state';
+import authContext from '../state/state';
 import * as EmailValidator from 'email-validator';
 
 import '../styles/registro.css';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Why is this comment necessary?

Reply:
it's no necessary, but if I don't put it, eslint shows a error message